cms/search/tests.py

```diff
@@ -145,6 +145,20 @@ def test_date_range(self):
         )
         self.assertResultsRangeSummaryEquals(response, min=1, max=7, total=7)
 
+    def test_date_range_remembers_date(self):
+        """If the dates were filled in before the user clicked the search button,
+        we fill in the dates afterwards. (Also demonstrates the differing date ranges)"""
+        response = self.client.get(
+            "/search/?query=e&content_type=all&after-year=2021&before-year=2022"
```

Comment:
For completeness I'd like it if we could have another test which includes all the date parameters (with a non-1st-January and non-31-December value) so we can test that this is correctly passing all of them through

Reply:
Thanks! Done.